package.json

```
@@ -103,7 +103,7 @@
 		"jsonwebtoken": "^8.5.1",
 		"lodash": "^4.17.21",
 		"marked": "^0.7.0",
-		"mathjax": "2.7.4",
+		"mathjax": "^2.7.4",
```

```suggestion
		"mathjax": "^2.7.8",
```
Maybe we can make the update more explicit?

---

```suggestion
		"mathjax": "^2.7.9",
```